elastic/kibana
# [ObsUX][Profiling, Infra] Add educational banners and tooltips

Closes https://github.com/elastic/kibana/issues/173269

## Summary

Adds educational banners and tooltips to the Profiling tab in Infra.

https://github.com/elastic/kibana/assets/793851/f5a19fd1-4900-4f35-aa8c-f35230238fe4

## How to test

1. Refer to docs to setup profiling [locally](https://github.com/elastic/observability-dev/blob/main/docs/profiling/on_prem.md#setting-profiling-on-prem) or [connected to oblt cluster](https://github.com/elastic/observability-dev/blob/main/docs/profiling/cloud.md#setting-up-local-kibana-to-oblt-clusters)
2. Open one of the hosts and make sure that you see the new banner and tooltips in the tabs.
3. Make sure the banner is dismissible and it's state is preserved when you refresh the page
